src/Cake.Common/Tools/NuGet/Pack/NuSpecDependency.cs

```diff
@@ -21,6 +21,19 @@ public class NuSpecDependency
         /// <value>The dependency's version.</value>
         public string Version { get; set; }
 
+        /// <summary>
+        /// Gets or sets a comma-delimited list of include/exclude tags indicating of the dependency to
+        /// include in the final package. The default value is <c>all</c>.
+        /// </summary>
+        public string Include { get; set; }
```

Comment:
I wonder if it would be nicer to have this as an IList<string> abstraction over the comma-separated values, rather than just expose the raw string itself?

Reply:
Yes would be better. I changed it.